src/pages/preferences/preferencesTabs/cliTab/VerifyCard.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Body } from "@leafygreen-ui/typography";
+import styled from "@emotion/styled";
+import { SiderCard } from "components/styles";
+import Code from "@leafygreen-ui/code";
+
+const { gray } = uiColors;
+
+export const VerifyCard = () => {
+  const verifcationCode = `
+"[evergreen] 2020/05/01 13:30:02 [p=notice]:
```

Comment:
Is there a ticket to query for the right version from the server?

Reply:
https://jira.mongodb.org/browse/EVG-8076